components/Docs/index.jsx

```diff
@@ -45,10 +45,11 @@ Request
 ~~~json
 {
     "prompt": "The request prompt goes here",
-    "tool": "The tool we want the off-chain agent to use goes here, for example openai-gpt4",
+    "tools": ["The tools we want to use go here", "can be multiple or just one"],
     "nonce": "The uuid goes here"
 }
 ~~~
+If the request specifies tools that the Agent has specified in its metadata (also on IPFS) then the request is valid and the Agent will perform the task. If the request uses tools that the Agent has not specified in its metadata then the request is invalid and the Agent will not perform the task.
```

Comment:
Please provide a sample of the agent metadata file. Use info from here too: https://aimechs.autonolas.network/

Reply:
See also the metadata generator on the factory. It should all link up. I think we're still missing the tool list there

Reply:
Same applies to that form - coordinate with @mohandast52 to get the changes implemented